python_ta/__main__.py

```diff
@@ -1,11 +1,20 @@
-import os
 import sys
+from os import path
 from typing import List, Optional
 
 import click
 
 from python_ta import __version__, check_all, check_errors
 
+# Gives absolute path to source root directory (pyta)
+SOURCE_ROOT = path.normpath(path.join(path.dirname(__file__), ".."))
+
+# Defines relative path to the default config directory (python_ta/config)
+CONFIG_LOCATION = path.join("python_ta", "config")
```

Comment:
Please just add one variable `DEFAULT_CONFIG_LOCATION` that stores the value `config/.pylintrc`. I.e., it should be relative to this file. I asked you to add this constant to the `python_ta/config` subfolder: I don't think you understood what I meant, but the best place to add it is the `__init__.py` file. That way you can import it from `python_ta.config`.

Remove the other variables, they aren't necessary.

Reply:
Would you want me to change lines like these in the `python_ta/config/__init__.py` file?

Reply:
No, don't change these lines. You're correct that they're looking in a `config` directory, but the file names are not the same in the two branches. Not in scope for this PR.